packages/store/src/storeHookTypes.sol

```diff
@@ -0,0 +1,10 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.0;
+
+uint8 constant BEFORE_SET_RECORD = 1 << 0;
+uint8 constant AFTER_SET_RECORD = 1 << 1;
+// TODO: do we need to differentiate between static and dynamic set field?
```

Comment:
cross posting here from discord for context: the "set field" hook is soon replaced with a "splice" hook (see #1510)